app/src/main/res/values-in/sura_names.xml

```diff
@@ -0,0 +1,238 @@
+    <item>Al-Hijr</item>
+    <item>An-Nahl</item>
+    <item>Al-Isra\'</item>
+    <item>Al-Kahfi</item>
+    <item>Maryam</item>
+    <item>Tha-Ha</item>
+    <item>Al-Anbiya\'</item>
+    <item>Al-Hajj</item>
+    <item>Al-Mu\'minun</item>
+    <item>An-Nur</item>
+    <item>Al-Furqan</item>
+    <item>Asy-Syu\'ara</item>
+    <item>An-Naml</item>
+    <item>Al-Qashash</item>
+    <item>Al-Ankabut</item>
+    <item>Ar-Rum</item>
+    <item>Luqman</item>
+    <item>As-Sajdah</item>
+    <item>Al-Ahzab</item>
+    <item>Saba\'</item>
+    <item>Fathir</item>
+    <item>Ya-Sin</item>
+    <item>Ash-Shaffat</item>
+    <item>Shad</item>
+    <item>Az-Zumar</item>
+    <item>Ghafir</item>
+    <item>Fushilat</item>
+    <item>Asy-Syura</item>
+    <item>Az-Zukhruf</item>
+    <item>Ad-Dukhan</item>
+    <item>Al-Jatsiyah</item>
+    <item>Al-Ahqaf</item>
+    <item>Muhammad</item>
+    <item>Al-Fath</item>
+    <item>Al-Hujurat</item>
+    <item>Qaf</item>
+    <item>Adz-Dzariyat</item>
+    <item>Ath-Thur</item>
+    <item>An-Najm</item>
+    <item>Al-Qamar</item>
+    <item>Ar-Rahman</item>
+    <item>Al-Waqi\'ah</item>
+    <item>Al-Hadid</item>
+    <item>Al-Mujadilah</item>
+    <item>Al-Hasyr</item>
+    <item>Al-Mumtahanah</item>
+    <item>Ash-Shaf</item>
+    <item>Al-Jumu\'ah</item>
+    <item>Al-Munafiqun</item>
+    <item>At-Taghabun</item>
+    <item>Ath-Thalaq</item>
+    <item>At-Tahrim</item>
+    <item>Al-Mulk</item>
+    <item>Al-Qalam</item>
+    <item>Al-Haqqah</item>
+    <item>Al-Ma\'arij</item>
+    <item>Nuh</item>
+    <item>Al-Jinn</item>
+    <item>Al-Muzzammil</item>
+    <item>Al-Muddatstsir</item>
+    <item>Al-Qiyamah</item>
+    <item>Al-Insan</item>
+    <item>Al-Mursalat</item>
+    <item>An-Naba\'</item>
+    <item>An-Nazi\'at</item>
+    <item>\'Abasa</item>
+    <item>At-Takwir</item>
+    <item>Al-Infithar</item>
+    <item>Al-Muthaffifin</item>
+    <item>Al-Insyiqaq</item>
+    <item>Al-Buruj</item>
+    <item>Ath-Thariq</item>
+    <item>Al-A\'la</item>
+    <item>Al-Ghasyiyah</item>
+    <item>Al-Fajr</item>
+    <item>Al-Balad</item>
+    <item>Asy-Syams</item>
+    <item>Al-Lail</item>
+    <item>Adh-Dhuhaa</item>
+    <item>Al-Insyirah</item>
+    <item>At-Tin</item>
+    <item>Al-\'Alaq</item>
+    <item>Al-Qadar</item>
+    <item>Al-Bayyinah</item>
+    <item>Az-Zalzalah</item>
+    <item>Al-\'Adiyat</item>
+    <item>Al-Qari\'ah</item>
+    <item>At-Takatsur</item>
+    <item>Al-\'Ashr</item>
+    <item>Al-Humazah</item>
+    <item>Al-Fil</item>
+    <item>Quraisy</item>
+    <item>Al-Ma\'un</item>
+    <item>Al-Kautsar</item>
+    <item>Al-Kafirun</item>
+    <item>An-Nashr</item>
+    <item>Al-Lahab</item>
+    <item>Al-Ikhlas</item>
+    <item>Al-Falaq</item>
+    <item>An-Nas</item>
+  </string-array>
+
+  <!-- If sura name has no translation, please add empty double quotes to the
+       respective item body to keep Android Studio happy -->
+  <string-array name="sura_names_translation">
```

Comment:
You do not seem to be translating sura names into Indonesian at all. Please remove this array.